app/views/shared/check_answers/_merits.html.erb

```diff
@@ -246,6 +246,23 @@
             ) %>
       </dl>
     <% end %>
+    <% if proceeding.prohibited_steps %>
+      <dl class="govuk-summary-list govuk-!-margin-bottom-0 print-remove-border">
```

Comment:
Are we manually writing the design system components or should we be using the `govuk-components` view library helpers?

Reply:
There were 15 already on the page so I went with consistency! 😄 
A ticket to review for component review would be good :+1: